tools/make/helm.mk

```diff
@@ -28,9 +28,15 @@ helm-install: helm-generate ## Install envoy gateway helm chart from OCI registr
 
 .PHONY: helm-generate
 helm-generate:
-	ImageRepository=${IMAGE} ImageTag=${TAG} ImagePullPolicy=${IMAGE_PULL_POLICY} envsubst < charts/gateway-helm/values.tmpl.yaml > ./charts/gateway-helm/values.yaml
+	GatewayImage=${IMAGE}:${TAG} GatewayImagePullPolicy=${IMAGE_PULL_POLICY} envsubst < charts/gateway-helm/values.tmpl.yaml > ./charts/gateway-helm/values.yaml
 	helm lint charts/gateway-helm
 
-helm-template: ## Template envoy gateway helm chart.
+HELM_VALUES := $(wildcard test/helm/*.in.yaml)
+
+helm-template: ## Template envoy gateway helm chart.z
```

Comment:
imo lets create a new `helm-test` target instead of reusing this one

Reply:
it not a test, because it will alwasy pass. it just help code review.